nuntium/templates/nuntium/profiles/messages_per_instance.html

```diff
@@ -81,8 +101,19 @@
       {% paginate %}
 
 <div class="modal fade" id="modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
-  <div class="modal-dialog modal-lg">
-    <div class="modal-content b-color">
+  <div class="modal-dialog">
+    <div class="modal-content">
+     <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title">{% trans "Delete this message?"%}</h4>
+      </div>
+      <div class="modal-body">
+        <p>{% trans "Are you sure you want to delete this message? This can't be undone."%}</p>
```

Comment:
Probably as discussed in #611 we wouldn't need to delete a message just turn it 'private'.